app/frontend/app/templates/group/manage.hbs

```diff
@@ -0,0 +1,20 @@
+<div class="container">
+  <div class="col-sm-12">
+    {{#if currentMember.isAdmin}}
+        <div class="panel panel-default">
+          <div class="panel-heading">
+            Group Settings
+          </div>
+          <div class="panel-body">
+            {{#if model.closed}}
+              Your group is closed! You'll need to contact a staff member to re-open it.
+            {{else}}
+              <button class="btn btn-danger" {{action "closeGroup"}}>
+                Close Group
+              </button>
```

Comment:
Shouldn't this be removed anyways? I though this action would be restricted to `currentUser.isAdmin` instead of `currentMember.isAdmin` now.